content/images/logos/windows.svg

```diff
@@ -1,14 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C/DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg
-   xmlns="http://www.w3.org/2000/svg" 
-   version="1.1"
-   height="88"
-   width="88">
-   
-   <path
-      fill="#00adef"
-      d="m0,12.402,35.687-4.8602,0.0156,34.423-35.67,0.20313zm35.67,33.529,0.0277,34.453-35
-      67-4.9041-0.002-29.78zm4.3261-39.025,47.318-6.906,0,41.527-47.318,0.37565zm47.329,39.
-      49-0.0111,41.34-47.318-6.6784-0.0663-34.739z" />
-</svg
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" height="88" width="88">
+<path style="fill:#00adef;" d="m0,12.402,35.687-4.8602,0.0156,34.423-35.67,0.20313zm35.67,33.529,0.0277,34.453-35.67-4.9041-0.002-29.78zm4.3261-39.025,47.318-6.906,0,41.527-47.318,0.37565zm47.329,39.349-0.0111,41.34-47.318-6.6784-0.0663-34.739z"/>
```

Comment:
You should replace the style with a `fill="#00adef"`, so that it doesn’t use inline CSS.

Reply:
everything should be fixed, it just took too long because I just cba to clone repos for such "simple" issues so I work on the web and fix changes on the fly.